server/modules/statisticalFactsheets/manager.js

```diff
@@ -0,0 +1,93 @@
+const R = require('ramda')
+const CarbonStockRepository = require('../carbonStock/repository/carbonStock')
+const ExtentOfForestRepository = require('../extentOfForest/repository/extentOfForestView')
+const ForestCharacteristicsRepository = require('../forestCharacteristics/repository/forestCharacteristicsView')
+const GrowingStockRepository = require('../growingStock/repository/growingStockTotalView')
+const SpecificForestCategoriesRepository = require('../specificForestCategories/repository/specificForestCategories')
+const ForestAreaWithinProtectedAreasRepository = require('../forestAreaWithinProtectedAreas/repository/forestAreaWithinProtectedAreas')
+const ForestOwnershipRepository = require('../forestOwnership/repository/forestOwnership')
+const PrimaryDesignatedManagementObjectiveRepository = require('../primaryDesignatedManagementObjective/repository/primaryDesignatedManagementObjective')
+const countryConfig = require('../../country/countryConfig')
+
+const utils = require('./utils')
+
+const KEYS = {
+  faoStat: 'faoStat',
+  area: 'area',
+  totalArea: 'totalArea',
+}
+
+const _getTotalArea = ({ countryIso, year }) => R.pathOr(0, [countryIso, KEYS.faoStat, year, KEYS.area], countryConfig)
+
+const getExtentOfForest = async (schemaName = 'public') => {
```

Comment:
is the defaultValue schema needed? isn't it required and always passed by the caller ?